static/xref/script.js

```diff
@@ -216,6 +201,25 @@ async function ready() {
     handleSubmit();
   }
 
+  // set up Advanced Search toggle
+  /** @type {HTMLInputElement} */
+  const advancedSearchToggle = form.querySelector("input[name='advanced']");
+  advancedSearchToggle.onchange = () => {
+    form.querySelectorAll('.advanced').forEach(input => {
+      input.hidden = !advancedSearchToggle.checked;
```

Comment:
``` diff
- input.hidden = !input.hidden;
+ input.hidden = !advancedSearchToggle.checked;
```
To keep it in sync with checkbox state, instead of toggling. (and removed `!advancedSearchToggle.checked && ` using that)

Reply:
Yeah, much better